comp/forwarder/defaultforwarder/resolver/domain_resolver.go

```diff
@@ -33,38 +38,98 @@ type DomainResolver interface {
 	// Resolve returns the domain to be used to send data for a given `transaction.Endpoint` along with a
 	// destination type
 	Resolve(endpoint transaction.Endpoint) (string, DestinationType)
+	// GetEndpoints returns the list of endpoints associated with this `DomainResolver`
+	GetEndpoints() []utils.APIKeys
```

Comment:
This doesn't returns endpoints but the API keys used by this resolver, no ? Should we rename this to `GetAPIKeysInfo` or something similar ?